### DISCUSSION TITLE
docs/v1.11.x/start/prepare/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#2591] [Announcements]
# docs/v1.11.x/start/prepare/

https://kubeovn.github.io/docs/v1.11.x/start/prepare/

**Comment by kinbod** [comment]
sealos 部署1.11.3 pinger一直卡住部署失败

**Reply by oilbeater** [reply]
问题的话请开个 issue 并贴上对应的日志

**Comment by Git4Mark** [comment]
请问kube-ovn可以给pod添加路由表吗，有计划支持这种功能吗

**Reply by oilbeater** [reply]
暂时没有计划
